### PR TITLE
refactor: template refactor

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |
| Chore            | ✔/✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]
🥇 Good Job

File: src/common/providers/fSProvider.ts [hunk 1]
```diff
@@ -19,7 +20,7 @@ export class FSProvider implements IConfigProvider {
       await replaceYamlFileContent(this.fsConfig.yamlFilePath, yamlContent);
     } catch (error) {
       // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-      this.logger.log('error', `Failed to update file: ${error}`);
+      this.logger.error(`Failed to update file: ${error}`);
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply src/common/providers/fSProvider.ts:23]: fixed

File: package.json [hunk 2]
```diff
@@ -1,14 +1,15 @@
 {
-  "name": "mapproxy-api",
-  "version": "1.4.3",
+  "name": "service-name",
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply package.json:2]: fixed

File: openapi3.yaml [hunk 3]
```diff
@@ -22,6 +22,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error'
+        409:
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply openapi3.yaml:25]: fixed

File: helm/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: mapproxy-api
+description: A Helm chart for mapproxy-api service
+type: application
+version: 1.0.0
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/Chart.yaml:5]: its the helm version, same as in overseer and etc.. its not the service version

File: README.md [hunk 5]
```diff
@@ -1,136 +1,126 @@
-# Map Colonies - MapProxy API Service
+# Map Colonies typescript service template
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply README.md]: Fixed

File: helm/certs/avi.crt [hunk 6]
```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```
Comment (asafMasa) [inline helm/certs/avi.crt:1]: is it suppose to be in this PR?

File: helm/certs/avi.key [hunk 7]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (asafMasa) [inline helm/certs/avi.key:1]: is it suppose to be in this PR?

File: helm/certs/rootCA.crt [hunk 8]
```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```
Comment (asafMasa) [inline helm/certs/rootCA.crt:1]: is it suppose to be in this PR?

File: helm/local.yaml [hunk 9]
```diff
@@ -0,0 +1,119 @@
+global:
```
Comment (asafMasa) [inline helm/local.yaml:1]: local should be ignore?

File: package.json [hunk 10]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "service-name",
+  "name": "mapproxy-api",
   "version": "1.0.0",
```
Comment (asafMasa) [inline package.json:3]: version shuld be the same as was published before

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

**Pull request merged**